engines/qdengine/qdcore/qd_game_object_moving.h

```diff
@@ -45,6 +45,7 @@ class qdGameObjectMoving : public qdGameObjectAnimated {
 	}
 
 	enum movement_mode_t {
+		MOVEMENT_MODE_NONE_EARLY,
```

Comment:
This is wrong, you shifted IDs, and they get saved /loaded(see qdGameObjectMoving::save_data()). E.g. put it to the enum end.

also, please check  `static const char *movementList[]`